fire/api/firedb/luk.py

```diff
@@ -55,21 +55,33 @@ def luk_punkt(self, punkt: Punkt, sagsevent: Sagsevent, commit: bool = True):
         self._luk_fikspunkregisterobjekt(punkt, sagsevent, commit=False)
 
         for geometriobjekt in punkt.geometriobjekter:
+            if geometriobjekt.registreringtil is not None:
```

Comment:
Du kan med fordel indsætte en kommentar omkring linje 57, der forklarer årsagen til at objekter hvor `registreringtil is not None` springes over. Jeg forstod det først da jeg læste commit-beskeden, men det er ikke sikkert jeg husker at kigge der næste gang jeg skal læse den her stump kode. Men derudover giver det jo ganske god mening at vi ikke skal overskrive eksisterende historik. Godt fanget!

Reply:
Så er det rettet og har lavet lidt om i funktionen så der nu kun er ét loop over alle objekterne som skal lukkes. Når nu jeg var i gang har jeg også rettet stavefejlen i navn på _luk_fikspunk**t**registerobjekt .

Der er stadig en lille opgave i at få ryddet op i det som allerede er kommet i databasen. Har en liste over lign. småting som vi kan tage på et tidspunkt.

Reply:
Fin løsning. Og godt at få rettet stavefejlen!